.github/workflows/ci-cd.yml

```
@@ -116,26 +116,38 @@ jobs:
         with:
           name: build-files
           path: artifact-bundle
+          retention-days: 1
 
   docker:
     runs-on: ubuntu-latest
     needs: build
     if: github.event_name == 'push' && github.ref == 'refs/heads/main'
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)